### PR TITLE
Add DESTDIR to command line arguments

### PR DESCRIPTION
**Pull request by izgzhen**

### CODE REVIEW

**Comment by izgzhen** [comment]
After that, I can conveniently setup an `install` test in http://hadrian-status-dev.zhenzhang.me

File: src/Settings/Packages/Rts.hs [hunk 1]
```diff
@@ -62,6 +62,7 @@ rtsPackageArgs = package rts ? do
     ffiIncludeDir  <- getSetting FfiIncludeDir
     ffiLibraryDir  <- getSetting FfiLibDir
     ghclibDir      <- expr installGhcLibDir
+    destDir        <- expr getDestDir
```
Comment (snowleopard) [inline src/Settings/Packages/Rts.hs]: Looks like this `destDir` is unused?

Comment (snowleopard) [inline src/Settings/Packages/Rts.hs]: Ah, it is used! Disregard this comment.

**Comment by snowleopard** [comment]
@izgzhen Thanks, looks good! I might do some further minor refactoring later, but I'll merge this now.

**Pull request merged**